### PR TITLE
doc,opensearch: add troubleshooting for opensearch-dashboards

### PR DESCRIPTION
**Pull request by RommelLayco**
# What changed, and why it matters

Add explaination on why opensearch-dashboards becomes unavailable when there is minor version mismatch

[SRE-6859]

[SRE-6859]: https://aiven.atlassian.net/browse/SRE-6859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mteiste** [comment]
The explanation is correct. Not qualified to review it from the style/language point of view.

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Troubleshoot opensearch dashboards
+==================================
+
+OpenSearch® Dashboards is unavailable
+---------------------------------------
```
Comment (harshini-rangaswamy) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:5]:
```suggestion
OpenSearch® Dashboards are unavailable
-------------------------------------------
```

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+OpenSearch® Dashboards requires the version of OpenSearch® on your cluster to be the same version. If OpenSearch® Dashboards is unavailable
+
+You may see the following logs in your opensearch-dashboards
```
Comment (harshini-rangaswamy) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:9]: OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the opensearch-dashboards logs:

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+```
```
Comment (harshini-rangaswamy) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:10]: I would suggest adding this in code style, with a line space above.

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+This version of OpenSearch Dashboards (v1.3.2) is incompatible with the following OpenSearch nodes in your cluster: v1.2.4 @ opensearch-searchdex-1.aiven.local/[${ipaddress}]:9200 (${ip_address}
+```
+This situation typically occurs when there is a new version of opensearch released.
```
Comment (harshini-rangaswamy) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:13]:
```suggestion
This situation typically occurs when a new version of OpenSearch is released.
```

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+When a new version of opensearch is available, any new nodes that are created for the service will use the new version of opensearch. The aiven platform will detect that the nodes in the service have different version and will begin upgrading the old nodes to a new version. Once this upgrade has been completed opensearch-dashboards should become available.
```
Comment (harshini-rangaswamy) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:15]:
```suggestion
When a new version of OpenSearch is available, any newly created nodes for the service will automatically use the latest version. The Aiven platform will identify any variance in versions among the service nodes and initiate the process of upgrading outdated ones to the new version. Following the successful completion of this upgrade, Opensearch-dashboards will be accessible.
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
@RommelLayco, thank you for adding this. I have suggested some changes to the text.

File: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Troubleshoot OpenSearch® Dashboards
+==================================
+
+OpenSearch® Dashboard is unavailable
+---------------------------------------
+
+OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the OpenSearch Dashboards logs:
+
+.. code:: text
```
Comment (RommelLayco) [inline docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst:9]: is this what you mean by using a code style. When i try to preview it on github. it does not look like it is getting formatted. 

Also how do ignore these vale errors? as this is what the log message looks like.

Comment (harshini-rangaswamy) [reply docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst]: @RommelLayco 
I included it in a code-block. Your approach is also accurate; when formatting after "..code::", ensure to create a line break and indent the text.
I noticed that the Table of Contents (TOC) entry was missing for this topic. I've included the TOC entry and updated the code formatting accordingly.

**Review by harshini-rangaswamy — APPROVED** [review]

**Pull request merged**